core/src/test/java/org/apache/iceberg/rest/auth/TestOAuth2Manager.java

```diff
@@ -509,7 +509,8 @@ void tableSessionDisallowedTableProperties() {
 
   @Test
   void standaloneTableSessionEmptyProperties() {
-    Map<String, String> properties = Map.of();
+    Map<String, String> properties =
+        Map.of(OAuth2Properties.OAUTH2_SERVER_URI, "https://auth-server.com/v1/token");
```

Comment:
I guess this can be reverted, since the test says "empty properties" and we should be falling back to the tokens endpoint if the property isn't specified